CHANGELOG.md

```diff
@@ -347,6 +347,9 @@ Library management
 
   Previously such `.agda-lib` files were ignored.
 
+* Now supports reading files with extension `.lagda.typ`, and use the parser for
```

Comment:
It is unclear to me to what extend Agda now supports `.typ`, esp. emacs mode / interaction, backends, so either the feature need to be completed or a description of the limitations be added.

Reply:
I'll add emacs mode support, hold on!

Backends are unsupported, as implied by "supports *reading*"

Reply:
It looks like we just need to put `(add-to-list 'auto-mode-alist '("\\.lagda.typ\\'" . agda2-mode))` in `.emacs`.

Reply:
How does it look like now?

Reply:
Weird, I was under the impression that the Agda emacs mode defines also the `lagda` extensions, but this is not the case: https://github.com/agda/agda/blob/d854cc9aebe80012c00ec75fa68ccad63efa57b4/src/data/emacs-mode/agda2.el#L11-L14
But, on a second thought, it makes sense that the user should switch manually modes for `.lagda.md` and related files.

In the light of that, maybe the `auto-mode-alist` should rather be written as a hint for the user who wants the agda-mode _autoloaded_ for the extension.